geopoint/challenges/default.json

```diff
@@ -337,6 +337,12 @@
           "iterations": 50,
           "tags": ["distance", "esql", "sort"]
         },
+        {
+          "operation": "distanceSort-esql-partial",
```

Comment:
For my curiosity, why do we have different warmups and iterations in these new benchmarks and the rationally behind this numbers?

Reply:
I chose numbers that are related to the inverse of the latency. So a very long running benchmark (like the 10-20 seconds of the sort before the performance) should have very few iterations (we really don't care about precision on such long queries), while fast benchmarks (10-50ms) usually have high iteration counts, so we can get more stable, and predictable results. But in general I've also started moving away from really high numbers like 200/100 because they do not seem more stable than numbers like 50/50. I think variability is coming from the cloud infrastructure, so high iteration counts are costing time and money without bringing value.